### DISCUSSION TITLE
无人认领的插件清单

### DISCUSSION THREAD

**Discussion by MKStoler4096** [ClassIsland/ClassIsland#344] [常规]
# 插件（非程序主功能）

- [ ] 待办提醒 #246
- [ ] 开机自动根据表格更换课表 #241
- [ ] 快速换课面板 #192
- [ ] 值日表 #159
- [ ] 添加下课准备提醒 #126
- [ ] 大字精确到秒的倒计时 #68
- [ ] 类似于`TimerIn`一样的下课需要手动关闭的提醒窗口 #54
- [ ] 考试模式 #314
- [ ] 设置密码 #311
- [ ] 分贝检测组件 #363

**Comment by DryIce-cc** [comment]
#192 一直都有人认领，不过还没做而已（）

**Reply by MKStoler4096** [reply]
那也约等于没人认领（Bushi
